docker/dev/find_fastest_apt_source.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Define a list of mirrors without using Bash arrays.
+# c.f. https://stackoverflow.com/a/23930212/724872
+read -r -d '' APT_SOURCES << EOM
+mirrors.aliyun.com
+mirrors.ustc.edu.cn
+mirrors.163.com
+archive.ubuntu.com
+ftp.de.debian.org
+packages.debian.org
+EOM
+
+# Find the fastest APT source using ping.
+SPEED=99999.9
+for i in $APT_SOURCES; do
+    # c.f. https://stackoverflow.com/a/9634982/724872
+    echo "Testig speed of $i ..."
+    CUR_SPEED=$(ping -c 4 $i | tail -1| awk '{print $4}' | cut -d '/' -f 2)
+    echo "$CUR_SPEED"
+
+    # c.f. https://stackoverflow.com/a/31087503/724872
+    if (( $(echo "$CUR_SPEED < $SPEED" | bc -l) )); then
+        BEST_APT_SOURCE="$i"
+        SPEED="$CUR_SPEED"
+    fi
+done
+
+# The default Ubuntu version is 18.04, code named bionic.
+UBUNTU_CODENAME=${UBUNTU_CODENAME-"bionic"}
+
+# Write APT source lists.
+cat <<EOF
```

Comment:
should write to /etc/apt/sources.list

Reply:
Let us write to the file at invocation time, like explained in https://github.com/sql-machine-learning/sqlflow/pull/2342#issue-422525337, so could we add Bash unit test for this script in the future.